src/components/Lookup/readme.md

```diff
@@ -47,7 +47,7 @@
         }
     }
 
-    initialState = { options: null };
+    initialState = { option: { label: 'Paris' }, options: null };
```

Comment:
the first example has to be the most simplest, this make the lookup to have an option selected by default, then make a new example with this and name it :
Lookup with initial option selected
Also put this new example at fourth position because now there is integrations tests based on the first 3 examples and this break tests